src/DynamoDbQueryBuilder.php

```diff
@@ -676,6 +676,22 @@ public function count()
 
         return $res['Count'];
     }
+    
+    public function pagedCount()
+    {
+        $limit = isset($this->limit) ? $this->limit : static::MAX_LIMIT;
+
+        $raw = $this->toDynamoDbQuery(['count(*)'], $limit);
+    
+        if ($raw->op === 'Scan') {
+            $res = $this->client->scan($raw->query);
+        } else {
+            $res = $this->client->query($raw->query);
+        }
+        $this->lastEvaluatedKey = Arr::get($res, 'LastEvaluatedKey');
+        
+        return (object)['count' => $res['Count'], 'scanned_count' => $res['ScannedCount'], 'lastKey' => $this->lastEvaluatedKey];
```

Comment:
```suggestion
        return ['count' => $res['Count'], 'scanned_count' => $res['ScannedCount'], 'lastKey' => $this->lastEvaluatedKey];
```

I think you can return the associative array directly.

Reply:
Or create a specific class for the object (and update the `@return`).

Reply:
This will need update to readme.

Reply:
I prefer the class option. 👀